client/src/app/app.component.html

```diff
@@ -1,4 +1,8 @@
 <mat-sidenav-container>
+  <mat-toolbar class="banner" color="warn">
```

Comment:
Huh material.

Reply:
yes this application is almost 6 years old now. One of the pushes when we migrate to the new api is to redo the ui as well.